apps/gidd/schema.py

```diff
@@ -208,6 +208,28 @@ class GiddYearType(graphene.ObjectType):
     year = graphene.Int(required=True)
 
 
+class GiddEventAffectedCountryType(graphene.ObjectType):
+    iso3 = graphene.String(required=True)
+    country_name = graphene.String(required=True)
+    new_displacement = graphene.Int()
+
+
+class GiddEventType(graphene.ObjectType):
+    event_name = graphene.String(required=True)
+    new_displacement = graphene.Int()
+    start_date = graphene.Date(required=True)
+    end_date = graphene.Date(required=True)
+    glide_numbers = graphene.List(
+        graphene.NonNull(graphene.String),
+    )
```

Comment:
Maybe we can do something like this
```suggestion
    glide_numbers = graphene.List(
        graphene.NonNull(graphene.String),
        required=True,
    )
```

Reply:
@thenav56  I changed this.